sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -427,7 +428,38 @@ public String getDefaultLanguage() {
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<PiiEntityCollection> recognizePiiEntities(String document, String language) {
-        return recognizePiiEntityAsyncClient.recognizePiiEntities(document, language);
+        return recognizePiiEntityAsyncClient.recognizePiiEntities(document, language, null);
+    }
+
+    /**
+     * Returns a list of Personally Identifiable Information(PII) entities in the provided document
+     * with provided language code.
+     *
+     * For a list of supported entity types, check: <a href="https://aka.ms/tanerpii">this</a>.
+     * For a list of enabled languages, check: <a href="https://aka.ms/talangs">this</a>.
+     *
+     * <p><strong>Code sample</strong></p>
+     * <p>Recognize the PII entities details in a document with provided language code and RecognizePiiEntityOptions.
+     * Subscribes to the call asynchronously and prints out the entity details when a response is received.</p>
+     *
+     * {@codesnippet com.azure.ai.textanalytics.TextAnalyticsAsyncClient.recognizePiiEntities#string-string-RecognizePiiEntityOptions}
+     *
+     * @param document the text to recognize PII entities details for.
+     * For text length limits, maximum batch size, and supported text encoding, see
+     * <a href="https://docs.microsoft.com/azure/cognitive-services/text-analytics/overview#data-limits">data limits</a>.
+     * @param language The 2 letter ISO 639-1 representation of language. If not set, uses "en" for English as default.
+     * @param options The additional configurable {@link RecognizePiiEntityOptions options} that may be passed when
+     * recognizing PII entities.
+     *
+     * @return A {@link Mono} contains a {@link PiiEntityCollection recognized PII entities collection}.
+     *
+     * @throws NullPointerException if {@code document} is null.
+     * @throws TextAnalyticsException if the response returned with an {@link TextAnalyticsError error}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<PiiEntityCollection> recognizePiiEntities(String document, String language,
```

Comment:
did we miss this method in previous PII endpoint?

Reply:
^^ same question, could you update the API view to reflect these changes?

Reply:
This is the single document operation. Since we are introducing `RecognizePiiEntityOptions` we should add additional API for the single document operation. Just like what we did for Sentiment Analysis.

Reply:
ahh I see, you previously had one for only document and language.
Thanks for explaining Shawn!

Reply:
@samvaity  Due to the APIView upgrade, there is hard to see the diff comparing to last revision. 
https://apiview.dev/Assemblies/Review/942b652edaf44b7189f41270cf07e54f?diffRevisionId=b747ea33dc7c4128b1f10326fcb372a9